File: Arrays/One Dimensional Arrays/findUnique.cpp
#include <bits/stdc++.h>
using namespace std;

void findUnique(int arr[], int n){
    int ele = arr[0];
    for (int i = 1; i <= n; i++){
        
        //logic
        ele = ele ^ arr[i];
    }

    cout << ele;
}


int main(){
    int n;
    cin >> n;

    int arr[50];

    //taking array as input
    for (int i = 0; i <= n; i++){
        cin >> arr[i];
    }

    findUnique(arr, n);
}

File: Arrays/One Dimensional Arrays/findUnique.py
# taking array as input
arr = [int(x) for x in input().split()]


def findUnique(arr):
    l = len(arr)
    ele = arr[0]
    for i in range(1, l):

        #logic
        ele = ele ^ arr[i]

    return ele

print(findUnique(arr))
